Windows/Installer/ags.iss

```diff
@@ -111,6 +113,8 @@ Source: "Source\Linux\ags64"; DestDir: "{app}\Linux"; Flags: ignoreversion; Comp
 Source: "Source\Linux\lib32\*"; DestDir: "{app}\Linux\lib32"; Flags: ignoreversion recursesubdirs createallsubdirs; Components: linux
 Source: "Source\Linux\lib64\*"; DestDir: "{app}\Linux\lib64"; Flags: ignoreversion recursesubdirs createallsubdirs; Components: linux
 Source: "Source\Linux\licenses\*"; DestDir: "{app}\Linux\licenses"; Flags: ignoreversion recursesubdirs createallsubdirs; Components: linux
+; Web build components
+Source: "Source\Web\*"; DestDir: "{app}\Web"; Flags: ignoreversion; Components: web
```

Comment:
If the file names are fixed it is safer to explicitly list each component to make sure that the installer build fails if one of them is missing.

Reply:
Adjusted! Seems to be alright!